.gitignore

```diff
@@ -1,2 +1,4 @@
 build
 dist
+.idea
+.vscode
```

Comment:
I'd rather not add this kind of thing here.  If you use vscode you are probably better off adding this to your global `.gitignore`, no?